foreman/data_refinery_foreman/surveyor/test_end_to_end.py

```diff
@@ -36,7 +36,7 @@
 
 
 LOOP_TIME = 5  # seconds
-MAX_WAIT_TIME = timedelta(minutes=5)
+MAX_WAIT_TIME = timedelta(minutes=45)
```

Comment:
:angry: 45 minutes for one test?!?!?!? Have you tried finding an experiment with smaller and/or fewer samples that might run faster?